nitrite-mvstore-adapter/src/main/resources/META-INF/native-image/org.dizitart/nitrite-mvstore-adapter/serialization-config.json

```diff
@@ -76,5 +76,29 @@
     },
     {
         "name": "org.dizitart.no2.store.UserCredential"
+    },
+    {
```

Comment:
hints for the compat module - this should allow users to migrate v3 databases to v4 even when running as native image